src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -1,113 +1,89 @@
 
+{# Metadata and asset linking #}
+
 {%- block css %}
-    <script>
-      document.documentElement.dataset.mode = localStorage.getItem("mode") || "{{ default_mode }}";
-      document.documentElement.dataset.theme = localStorage.getItem("theme") || "light"
-    </script>
-    {{ _webpack.head_pre_assets() }}
-    {{ _webpack.head_pre_icons() }}
-    {{- css() }}
-    {{ _webpack.head_js_preload() }}
+<script>
+  document.documentElement.dataset.mode = localStorage.getItem("mode") || "{{ default_mode }}";
+  document.documentElement.dataset.theme = localStorage.getItem("theme") || "light"
+</script>
+{{ _webpack.head_pre_assets() }}
+{{ _webpack.head_pre_icons() }}
+{{- css() }}
+{{ _webpack.head_js_preload() }}
 {%- endblock %}
 
 {%- block extrahead %}
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="docsearch:language" content="{{ language }}">
-    {% for favicon in theme_favicons %}
-      {% if favicon.href[:4] == 'http'%}
-      <link rel="{{ favicon.rel }}" sizes="{{ favicon.sizes }}" href="{{ favicon.href }}">
-      {% else %}
-      <link rel="{{ favicon.rel }}" sizes="{{ favicon.sizes }}" href="{{ pathto('_static/' + favicon.href, 1) }}">
-      {% endif %}
-    {% endfor %}
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<meta name="docsearch:language" content="{{ language }}">
+{% for favicon in theme_favicons %}
+  {% if favicon.href[:4] == 'http'%}
+  <link rel="{{ favicon.rel }}" sizes="{{ favicon.sizes }}" href="{{ favicon.href }}">
+  {% else %}
+  <link rel="{{ favicon.rel }}" sizes="{{ favicon.sizes }}" href="{{ pathto('_static/' + favicon.href, 1) }}">
+  {% endif %}
+{% endfor %}
 
-    <!-- Google Analytics -->
-    {{ generate_google_analytics_script(id=theme_google_analytics_id) }}
+<!-- Google Analytics -->
+{{ generate_google_analytics_script(id=theme_google_analytics_id) }}
 {%- endblock %}
 
-{# Silence the sidebar's, relbar's #}
-{% block header %}{% endblock %}
-{% block relbar1 %}{% endblock %}
-{% block relbar2 %}{% endblock %}
-{% block sidebarsourcelink %}{% endblock %}
-
 {% block body_tag %}
   {# set up with scrollspy to update the toc as we scroll #}
   {# ref: https://getbootstrap.com/docs/4.0/components/scrollspy/ #}
   <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="180" data-default-mode="{{ default_mode }}">
 {%- endblock %}
+
 {%- block content %}
-    {# Added to support a banner with an alert #}
-    <div class="container-fluid" id="banner"></div>
+    <div class="bd-header-announcement container-fluid" id="banner">
+      {% include "sections/announcement.html" %}
+    </div>
 
     {% block docs_navbar %}
-    <nav class="navbar navbar-light navbar-expand-lg bg-light fixed-top bd-navbar" id="navbar-main">
-      {%- include "docs-navbar.html" %}
+    <nav class="bd-header-primary navbar navbar-light navbar-expand-lg bg-light fixed-top bd-navbar" id="navbar-main">
```

Comment:
Why is this class `bd-header-primary` and not `bd-header` as the section name and the image you shared in the PR ?

Reply:
good catch! the latest push renames this to just `header`!